src/App/App.jsx

```diff
@@ -0,0 +1,33 @@
+import "./App.css";
+import { Switch, Route } from "react-router-dom";
+import Header from "../Header";
+import Footer from "../Footer";
+import Home from "../pages/Home";
+import Cards from "../pages/Cards";
+
+function App() {
+    return (
+        <>
+            <Switch>
+                <Route path="/:path">
+                    <Header />
+                </Route>
+            </Switch>
```

Comment:
This is an interesting way to only render `Header` and `Footer` on non-index pages... if the Header component becomes more complicated and starts to differ by route/page type/something else we may want to have each route render its own `Header` instead

Reply:
ohhhhhhh so thats how it works :tsukasasparkle: